backend/routes/db.py

```diff
@@ -356,19 +357,7 @@ def _cset_members_items(codeset_ids: Union[str, None] = Query(default=''), ) ->
 @router.get("/db-refresh")
 def db_refresh_route():
     """Triggers refresh of the database"""
-
-    headers = {
-        "Authorization": f"Bearer {CONFIG['personal_access_token']}",
-        "Accept": "application/vnd.github.v3+json",
-        "Content-Type": "application/json"
-    }
-    # args = {"event-type":"refresh-db"}
-    url = 'https://api.github.com/repos/jhu-bids/TermHub/dispatches'
-    payload = {
-        'event_type': 'refresh-db',
-    }
-    response = requests.post(url, headers = headers,data=json.dumps(payload))
-    return response
+    os.system('python backend/db/refresh.py')
 
```

Comment:
@joeflack4 I added this to call refresh directly. I want to go with your suggestion of having fetch_cset_and_members_objects returning none as a way to check that there is no new data. After I change that I want to test this again to make sure it's working.

Reply:
Sounds good.

As for running the refresh script, what I would do here is import `refresh_db` and run that function directly. That should have the same effect as running the script as a process the way you're doing it, with the added benefits of (i) we can debug it if we need to, (ii) if there's a failure, we'll see the stacktrace, etc.

Also and FYI, concerning subprocesses, I haven't seen `os.system` before. I just tried it and I do see that it runs a system call. I've always used `subprocess.Popen` or `subprocess.call` (I prefer the latter). IDK about `os.system`, but you can get the results of these by like `results = subprocess.call()` which will allow you to see anything that was printed to the console while the process was running. It's a little wonkier than calling a function directly, but sometimes it's the only option.